src/protocols/rdp/settings.c

```diff
@@ -1529,7 +1549,20 @@ void guac_rdp_push_settings(guac_client* client,
         case GUAC_SECURITY_ANY:
             rdp_settings->RdpSecurity = TRUE;
             rdp_settings->TlsSecurity = TRUE;
-            rdp_settings->NlaSecurity = guac_settings->username && guac_settings->password;
+
+            /* Explicitly disable NLA if FIPS mode is enabled - it won't work */
+            if (guac_fips_enabled()) {
+
+                guac_client_log(client, GUAC_LOG_WARNING,
+                        "Disabling NLA security mode when FIPS mode is enabled.");
+                rdp_settings->NlaSecurity = FALSE;
+
+            }
+
+            /* Enable NLA security mode if both username and password are set */
+            else
+                rdp_settings->NlaSecurity = guac_settings->username && guac_settings->password;
```

Comment:
I think this is a holdover from back when we didn't support prompting for credentials and can be safely replaced with just `rdp_settings->NlaSecurity = TRUE;`.

Reply:
Ah ok